x-pack/legacy/plugins/siem/public/components/timeline/footer/index.tsx

```diff
@@ -327,44 +290,35 @@ export class Footer extends React.Component<FooterProps, FooterState> {
                   data-test-subj="paging-control"
                   hasNextPage={hasNextPage}
                   isLoading={isLoading}
-                  loadMore={this.loadMore}
+                  loadMore={loadMore}
                 />
               )}
             </EuiFlexItem>
 
             <EuiFlexItem data-test-subj="last-updated-container" grow={false}>
               <FixedWidthLastUpdated data-test-subj="fixed-width-last-updated" compact={compact}>
-                <LastUpdatedAt
-                  updatedAt={this.state.updatedAt || getUpdatedAt()}
-                  compact={compact}
-                />
+                <LastUpdatedAt updatedAt={updatedAt || getUpdatedAt()} compact={compact} />
               </FixedWidthLastUpdated>
             </EuiFlexItem>
           </FooterFlexGroup>
         </FooterContainer>
       </>
     );
+  },
+  (prevProps, nextProps) => {
+    return (
+      prevProps.compact === nextProps.compact &&
+      prevProps.hasNextPage === nextProps.hasNextPage &&
+      prevProps.height === nextProps.height &&
+      prevProps.isEventViewer === nextProps.isEventViewer &&
+      prevProps.isLive === nextProps.isLive &&
+      prevProps.isLoading === nextProps.isLoading &&
+      prevProps.itemsCount === nextProps.itemsCount &&
+      prevProps.itemsPerPage === nextProps.itemsPerPage &&
+      prevProps.itemsPerPageOptions === nextProps.itemsPerPageOptions &&
+      prevProps.serverSideEventCount === nextProps.serverSideEventCount
+    );
```

Comment:
Looking at the original when this was a class, these are the properties you're still missing here from the original shouldComponentUpdate for the compare of if this component should update:

<img width="498" alt="Screen Shot 2019-09-24 at 8 48 19 PM" src="https://user-images.githubusercontent.com/1151048/65565392-08664600-df0d-11e9-839f-b489c0452784.png">

```
isPopoverOpen
paginationLoading
updatedAt
```

Reply:
hmm... not sure how those got dropped? will put them in thanks

Reply:
this is because those were state updates, and this component is no longer using state but hooks. the logic that tracks the state updates is now in the `useEffect`